kernel/private/classes/ezscriptclusterpurge.php

```diff
@@ -65,6 +65,12 @@ public function run()
     {
         $cli = eZCLI::instance();
 
+        if ( empty( $this->optScopes ) )
```

Comment:
@gggeek : indeed, but if none is defined the default `false` is used.
So this PR just checks it as it doesn't intend any change in behavior (another possibility would be to use all)